### PR TITLE
@l2sucess => Edit actions/reducers

### PR DESCRIPTION
**Pull request by eessex**
- Duplicates the `editActions` and `editReducers` files as `editActions2` and `editReducers2` 
- Updates article actions to use article from store (rather than passing attributes to actions)

These files will replace the existing `editActions` and `editReducers` once the redux component changes are merged.

### CODE REVIEW

File: client/actions/test/editActions2.test.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    window.location.assign = jest.fn()
+    article = clone(FeatureArticle)
+  })
+
+  document.body.innerHTML =
+    '<div>' +
+    '  <div id="edit-sections-spinner" />' +
+    '  <input id="edit-seo__focus-keyword" value="ceramics" />' +
+    '</div>'
+
```
Comment (damassi) [inline client/actions/test/editActions2.test.js]: Can possibly do 

```javascript
document.body.innerHTML = `
  <div>
    <div id="edit-sections-spinner" />
    <input id="edit-seo__focus-keyword" value="ceramics" /> 
  </div>
`
```

File: client/reducers/editReducer2.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        section: null,
+        isSaved: false
+      }, state)
+    }
+
+    case actions.RESET_SECTIONS: {
+      const article = cloneDeep(state.article)
+      const { sections } = action.payload
+
+      extend(article, { sections })
```
Comment (damassi) [inline client/reducers/editReducer2.js]: This seems a little weird to mutate in place. Maybe find a way to assign it to variable and then pass it in?

Comment (eessex) [reply client/reducers/editReducer2.js]: That makes sense, can take care of it in the actions instead

Comment (damassi) [reply client/reducers/editReducer2.js]: Cool, yeah a good rule of thumb is to clone rather than mutate

**Pull request merged**